tests/data/rawvoice_mp4.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Namespace Test",
+  "episodes": [],
+  "mp4": ""
```

Comment:
It would probably make more sense to add this as files/enclodures, no? Does it even make sense to have the `mp4` field in a channel and not in an episode?